biobox_cli/behave_interface.py

```diff
@@ -60,7 +60,10 @@ def is_failed(behave_data):
     return "failed" in map(lambda i: i['status'], behave_data)
 
 def get_scenarios(behave_data):
-    return reduce(lambda acc, x: acc + x['elements'], behave_data, [])
+    acc = []
+    for item in behave_data:
+        acc += item['elements']
+    return acc
```

Comment:
`reduce` can be used with
```python
try:
    from functools import reduce
except ImportError:
    pass
```
I used this solution in another file, do you want me to use it here too?